lib/components/artist/shows/metadata.js

```diff
@@ -16,7 +17,7 @@ class Metadata extends React.Component {
         <Text style={styles.sansSerifText}>{this.showTypeString(this.props.show)}</Text>
         <SerifText style={styles.serifText}>{this.props.show.name}</SerifText>
         { this.dateAndLocationString(this.props.show) }
-
+        { this.statusText(this.props.show) }
```

Comment:
Btw, why are you passing the show to these methods instead of accessing `this.props.show` from within the method?